test/mail_ssl_manual.bats

```diff
@@ -109,6 +110,18 @@ function teardown_file() {
     assert_equal "${RESULT}" 'Verification: OK'
 }
 
+@test "checking ssl: manual cert changes are picked up by check-for-changes" {
+    printf 'someThingsChangedHere' \
+      >>"$(pwd)/test/test-files/ssl/${DOMAIN_SSL_MANUAL}/with_ca/ecdsa/key.ecdsa.pem"
```

Comment:
This parts actually plagued me a lot, because I saw weird issues with file permissions. While this is currently not ideal, its a functioning and practical solution. I'd rather not spend more time on this when you're about to rework this functionality anyways... (forgive my lazyness here, I was about to go crazy fiddling with the permission issues - so I came up with this solution).